README.md

```diff
@@ -1,6 +1,18 @@
 # TrackLooper
 
-## Quick start guide
+## Quick Start
+
+    git clone git@github.com:SegmentLinking/TrackLooper.git
+    cd TrackLooper/
+    source setup.sh
+    # source setup_cgpu.sh # if you are on cgpu-1
+    sdl_make_tracklooper -mc8
+    sdl -i PU200 -o LSTNtuple.root
+    createPerfNumDenHists -i LSTNtuple.root -o LSTNumDen.root
+    lst_plot_performance.py LSTNumDen.root -t "myTag"
```

Comment:
Can we suppress the output of this command. It is really long and probably no one will read it.

Reply:
with bin/sdl_run these are addressed.